frontend/src/containers/app/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { connect } from "react-redux"
+
+import logo from '../../logo.svg';
+import './App.scss';
+import { AppState, Auth } from '../../types';
+import { logIn } from '../../redux/auth/actions';
+
+interface Props {
+  auth: Auth
+  login: (email: string, password: string) => void
+}
+
+const App = React.memo((props: Props): JSX.Element => {
+  const [username, setUsername] = useState("")
+  const [password, setPassword] = useState("")
+  const onUsernameChange = (event: React.ChangeEvent<HTMLInputElement>):void => setUsername(event.target.value)
+  const onPasswordChange = (event: React.ChangeEvent<HTMLInputElement>):void => setPassword(event.target.value)
+
+  const onLoginClick = (): void => props.login(username, password)
+  return (
```

Comment:
Since this is a container, shouldn't the UI content go somewhere else?

Reply:
Made the change; In general yes but this was so small, it felt silly. But, as a pattern you're right, we should start and stay consistent.